.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,40 @@
+name: Golang Linter
+# Lint runs golangci-lint over the entire Tendermint repository.
+#
+# This workflow is run on every pull request and push to main.
+#
+# The `golangci` job will pass without running if no *.{go, mod, sum}
+# files have been modified.
+#
+# To run this locally, simply run `make lint` from the root of the repo.
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+  golangci:
+    name: golangci-lint
+    runs-on: ubuntu-latest
+    timeout-minutes: 8
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '1.18'
+      - uses: technote-space/get-diff-action@v6
+        with:
+          PATTERNS: |
+            **/**.go
+            go.mod
+            go.sum
+      - uses: golangci/golangci-lint-action@v3
+        with:
+          # Required: the version of golangci-lint is required and
+          # must be specified without patch version: we always use the
+          # latest patch version.
+          version: v1.49.0
+          args: --timeout 10m
+          github-token: ${{ secrets.PR_ACTION_TOKEN }}
```

Comment:
@kim201212 Would you mind explaining why we need `PR_ACTION_TOKEN` here, please?

Reply:
It needs to run `env.GIT_DIFF`

Reply:
😭

Reply:
Let's test without action token. I think it is not required.

Reply:
https://github.com/dsrvlabs/vatz/pull/311/commits/7143eca01df2159d95a2620aadfff2a4ae5de469

I think the GitHub token is not required 😢